rotkehlchen/chain/ethereum/modules/yearn/vaultsv2.py

```diff
@@ -189,6 +189,8 @@ def get_vaults_history(
                 query_addresses.append(EthAddress(address.lower()))
                 query_checksumed_addresses.append(address)
 
+        # Store maps for addresses and events retrieved from graph + database
+        events_mapping: Dict[str, List[YearnVaultEvent]] = {}
```

Comment:
for such maps what we do is provide a more descriptive name so that there is no need to guess what maps to what. i.e. `addresses_to_events`

Also why `str`? Shouldn't it be `ChecksumEthaddress`?

Reply:
Yes should be ChecksumEthaddress